src/modules/Common/components/ServicesList.tsx

```diff
@@ -0,0 +1,59 @@
+import Column from './Column';
+import Container from 'modules/Common/containers/Container';
+import { FiFileText as IconDocument } from 'react-icons/fi';
+import Link from 'next/link';
+import React from 'react';
+import { Services } from 'modules/OTA-api/api';
+import s from './ServicesList.module.css';
+
+type ServicesListProps = {
+  title: string;
+  subtitle: string;
+  services: Services;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+const ServicesList: React.FC<ServicesListProps> = ({
+  children,
+  className,
+  title,
+  subtitle,
+  services,
+  ...props
+}) => {
+  return (
+    <Container gridCols="12" gridGutters="11" className={s.listServices} {...props}>
+      <Column width={100} title={title} subtitle={subtitle}>
+        <ul className={s.listServices_items}>
+          {Object.keys(services || {}).map((serviceName: string) => {
+            const documentTypes = (services || {})[serviceName];
+            return (
+              <li className={s.listServices_item} key={`service_list_${serviceName}`}>
+                <div className={s.listServices_item_service}>{serviceName} :</div>
+                <div className={s.listServices_item_docs}>
+                  {documentTypes.map((documentType: string) => {
+                    const target = `https://github.com/ambanum/OpenTermsArchive-versions/blob/master/${serviceName}/${documentType}.md`;
```

Comment:
In fact Services List does not give me the information on what I'm looking at, which is the latest version recorded also



What would be more interesting would be to have all the latest changes
- for one service: https://github.com/ambanum/OpenTermsArchive-versions/commits/master/Apple
- for one document https://github.com/ambanum/OpenTermsArchive-versions/commits/master/Apple/Privacy%20Policy.md

And maybe all the snapshots or at least the last one

Maybe a table alignment would be thus more appropriated


Like so
Apple - link to "All versions" - link to "All snapshots"
| Document      | Latest version  | Latest snapshot  | All versions  | All snapshots  |
| ----------- | ----------- |----------- |----------- |----------- |
| Privacy Policy      | link       | link       |link       | link       |
| Terms of use      | link       | link       |link       | link       |
| Cookie Policy      | link       | link       |link       | link       |